src/main/scheduler/scheduler.h

```diff
@@ -243,6 +243,7 @@ void schedulerInit(void);
 void scheduler(void);
 timeUs_t schedulerExecuteTask(task_t *selectedTask, timeUs_t currentTimeUs);
 void taskSystemLoad(timeUs_t currentTimeUs);
+uint32_t getCPUPercentageLate(void);
```

Comment:
camelCasing abbreviations maybe makes them more readable.
```suggestion
uint32_t getCpuPercentageLate(void);
```